Core/Core/Extensions/WKWebViewExtensions.swift

```diff
@@ -28,6 +28,7 @@ extension WKWebView {
 
     public func handle(_ name: String, handler: @escaping MessageHandler) {
         let passer = MessagePasser(handler: handler)
+        configuration.userContentController.removeScriptMessageHandler(forName: name)
```

Comment:
I got a crash when the new window tried to add handlers that (apparently) already existed.